spmd/tensor/ops/dropout.py

```diff
@@ -28,3 +28,31 @@ def _dist_dropout(
                 mask, device_mesh=self.device_mesh, placements=self.placements
             ),
         )
+
+@register_impl("aten.native_dropout_backward.default")
```

Comment:
I know this is WIP, but can we use sharding propagation rule instead of register_impl? `register_impl` is only supposed to be used by custom written implementation. Example usage of sharding propagation rule https://github.com/pytorch/tau/blob/main/spmd/tensor/ops/math_ops.py#L34

Reply:
done, using the built-in delegation mechanisms now.